apps/bleprph_oic/syscfg.yml

```diff
@@ -25,6 +25,10 @@ syscfg.defs:
         value: 0
 
 syscfg.vals:
+    CONSOLE_MODE: full
```

Comment:
I do not recall if we have `CONSOLE_MODE` instead of `CONSOLE_IMPLEMENTATION` for a reason, but if it's just for 'historical reasons' we should add `CONSOLE_IMPLEMENTATION` and deprecate `CONSOLE_MODE`.

@kasjer ^^ ?

Reply:
I'm fine with renaming this to CONSOLE_IMPLEMENTATION to keep things consistent

Reply:
I would deprecate _MODE it was first attempt. Later on for logs _IMPLEMENTATION was chosen as it more clear what it does.